app/code/Magento/InventorySales/Api/ShippingAlgorithmInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Api;
+
+use Magento\Sales\Api\Data\OrderInterface;
+
+/**
+ * @api
+ */
+interface ShippingAlgorithmInterface
```

Comment:
Move into  `Inventory\Model\ShippingAlgorithm`  folder
Supposed that will be extension point (SPI) of concrete implementation but not part of API